### PR TITLE
Keep the error messages visible

### PR DESCRIPTION
**Pull request by mrsmkl**
Should help with #34

### CODE REVIEW

File: src/bin/contribute.rs [hunk 1]
```diff
@@ -305,7 +305,7 @@ impl Contribute {
                     Ok(false) => {}
                     Err(e) => {
                         warn!("Got error from updater: {}", e);
-                        progress_bar.set_message(&format!(
+                        progress_bar.println(&format!(
```
Comment (kobigurk) [inline src/bin/contribute.rs]: What happens if the message is long?
For the `set_message`, we used the format string to set it to not overflow and keep printing the same line over and over.
What will happen here?

File: src/bin/contribute.rs [hunk 2]
```diff
@@ -329,6 +329,7 @@ impl Contribute {
         for i in 0..total_tasks {
             let delay_duration = Duration::seconds(DELAY_AFTER_ERROR_DURATION_SECS).to_std()?;
             let mut cloned = self.clone_with_new_filenames(i);
+            let progress_bar2 = progress_bar3.clone();
```
Comment (kobigurk) [inline src/bin/contribute.rs]:
```suggestion
            let progress_bar_for_thread = progress_bar3.clone();
```

File: src/bin/contribute.rs [hunk 3]
```diff
@@ -339,6 +340,8 @@ impl Contribute {
                         Ok(_) => {}
                         Err(e) => {
                             warn!("Got error from run: {}, retrying...", e);
+                            progress_bar2
```
Comment (kobigurk) [inline src/bin/contribute.rs]:
```suggestion
                            progress_bar_for_thread 
```

File: src/bin/contribute.rs [hunk 4]
```diff
@@ -295,6 +294,7 @@ impl Contribute {
         let mut futures = vec![];
 
         let updater = self.clone();
+        let progress_bar3 = progress_bar.clone();
```
Comment (kobigurk) [inline src/bin/contribute.rs]:
```suggestion
        let progress_bar_for_thread = progress_bar.clone();
```

File: src/bin/contribute.rs (hunk 1 above)
Comment (kobigurk) [inline src/bin/contribute.rs]:
```suggestion
                        progress_bar_for_thread.println(&format!(
```

**Review by kobigurk — CHANGES_REQUESTED** [review]

File: src/bin/contribute.rs (hunk 1 above)
Comment (mrsmkl) [reply src/bin/contribute.rs]: It will just print the message once, but I guess it would get truncated.

Comment (kobigurk) [reply src/bin/contribute.rs]: The experience we had with `set_message` is that it wasn't truncated by default, it kept printing the same message over and over. I suggest we check that for println before committing to it.

Comment (mrsmkl) [reply src/bin/contribute.rs]: Here's what it looks like (I have a timer that calls `progress_bar.println` each 20 seconds):
```
Contributing! Please unmount and remove the USB drive containing your keys now.
sddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
sddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
sddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
⠐ [00:00:50] [####################] 8/8 Successfully contributed! Don't turn this off yet, you might still be needed. Thank you for participating!
```

Comment (kobigurk) [reply src/bin/contribute.rs]: OK, that looks good! So the overflow just prints the message once, and doesn't even truncate?

Comment (mrsmkl) [reply src/bin/contribute.rs]: Yeah, correct.

**Review by kobigurk — APPROVED** [review]

**Pull request merged**